docs/development.md

```diff
@@ -0,0 +1,150 @@
+# Development Process
+
+We develop and deploy on a 2-3 week schedule. Before a deploy, a new release is
+tagged and cut, deployment ticket filed, and QA verified. On occasion a release
+will get a full suite of load-tests as well.
+
+Some of the ancillary Push projects have no set release schedule and are
+released as needed.
+
+## Product Planning
+
+Product-level feature planning is done using OKR's, similar to
+[Google's OKR process][OKR]. Finalized OKR's per quarter result in feature
+cards being created in Aha, where they're then prioritized for work to begin. A
+single OKR may result in multiple feature cards depending on scope, each card
+is tagged for its corresponding OKR #.
+
+* [Aha Feature Cards](https://mozilla.aha.io/products/PUSHSVC/feature_cards)
+
+## Code Development
+
+Each Aha card being actively developed has a milestone created in the main
+[Mozilla Push-Service Repository][MPSM], which is then propagated out to all
+Push-related GitHub repositories.
+
+### Milestones & Waffleboard
+
+Issues are created in their appropriate repositories and associated with a
+milestone. Ownership and triage of issues is done via a single Waffleboard that
+aggregates issue tracking across all Push-related repos.
+
+* [Push Waffleboard](https://waffle.io/mozilla-services/push-service)
+
+All issues must have a milestone attached before assignment.
+
+### Labels
+
+Issues are assigned a priority based on the release the issue is targeted at.
+
+* P1 - Next release
+* P2 - Release after next
+* P3 - No immediate release targeted
+* P5 - No work planned yet
```

Comment:
the missing P4 always bothers me. It's like labeling three chickens 1, 2, and 4 letting them loose in an office building.